src/frontend/src/flows/loginUnknown.ts

```diff
@@ -99,6 +100,14 @@ export const loginUnknown = async (
   });
 };
 
+function isRegistrationAllowed() {
```

Comment:
a note here and some comments in `registerDisabled.ts` would be great!

Reply:
Done.